bin/linux-sh-installer

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# This is a self-extracting archive that installs Ink/Stitch on your Linux
+# system.  This first part is an installer, and after that is a .tar.zx file
+# containing Ink/Stitch itself.
+#
+# To install, simply run this script:
+#
+#   sh inkstitch-<version>.sh
+#
+#
+# EXPERT STUFF BELOW:
+#
+# If you'd rather install it yourself, run this script with --extract to
+# produce the original inkstitch-<version>.tar.xz file in the current
+# directory.
+#
+# This script will attempt to determine where to install Inkscape user
+# extensions automatically.  If it gets it wrong, you can set one of these
+# environment variables:
+#
+#   INKSCAPE_PATH 
+#     The path to the inkscape executable program.  This script will ask that program 
+#     where to install extensions by passing it the --user-data-directory argument.
+#
+#   INKSCAPE_EXTENSIONS_PATH
+#     The path to the inkscape extensions directory.  Use this to bypass the
+#     --user-data-directory method and specify a directory yourself.
+
+die() {
+  echo "$*"
+  exit 1
+}
+
+extract() {
+  ( grep -m1 '^__ARCHIVE__$' > /dev/null; cat ) < "$0"
+}
+
+find_inkscape() {
+  # allow expert user override
+  if [ -n "$INKSCAPE_PATH" ]; then
+    echo "$INKSCAPE_PATH"
+    return
+  fi
+
+  inkscape="$(which inkscape)"
+
+  if [ -z "$inkscape" ]; then
+    read -p "Please enter the path to the inkscape program executable (example: /usr/bin/inkscape): " inkscape
+  fi
+
+  if [ ! -x "$inkscape" ]; then
+    die "Inkscape not found or not executable ($inkscape)"
+  fi
+
+  echo "$inkscape"
+}
+
+find_extensions_dir() {
+  # allow expert user override
+  if [ -n "$INKSCAPE_EXTENSIONS_PATH" ]; then
+    echo "$INKSCAPE_EXTENSIONS_PATH"
+    return
+  fi
+
+  inkscape="$(find_inkscape)"
+
+  if [ -x "$inkscape" ]; then
+    extensions_dir="$(inkscape --user-data-directory)"
```

Comment:
This is the inkscape folder not the extensions folder. But it is used as the extensions folder. I think we should add /extensions here.

Reply:
Oh shoot, thank you!